scripts/config-loader.js

```diff
@@ -72,6 +72,7 @@ module.exports = function(source) {
   const template = Handlebars.compile(templateSource);
   outputFile = template({
     config: config,
+    languageCode: "en_US"
```

Comment:
Maybe I'm missing something, but wouldn't a user need to change this value to view the app in their chosen language?

For example, how would the user select to view the app in Spanish, and how would that update the languageCode here?

Reply:
So eventually I think we should move toward @zmbc's localize on the client idea, where we serve a single bundle with all localized content in it, and switching languages is just a matter of changing the `i18n` code. But as long as we still have content handled by gettext, we need to fetch a whole new localized bundle when the user switches languages.

So it's true that content in the above `messages.js` file that doesn't match the current locale is redundant. We could filter that out in the build process I guess, but I'm going on the assumption that eventually we'll want all language content available on the client.